src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -45,67 +45,58 @@ public ApplicationDeployer(DeploymentParameters deploymentParameters, ILogger lo
 
-                TargetRuntimeName = $"dnx-{flavor}-{GetOSPrefix()}-{architecture}.{currentRuntimeVersion}";
-                TargetRuntimeBinPath = Path.Combine(runtimeHome, TargetRuntimeName, "bin");
-            }
+            //// tooling runtime will stick to coreclr so as to prevent long path issue during publishing
+            //ToolingRuntimeBinPath = Path.Combine(runtimeHome, $"dnx-coreclr-{GetOSPrefix()}-{architecture}.{currentRuntimeVersion}", "bin");
 
-            if (!Directory.Exists(ToolingRuntimeBinPath) || !Directory.Exists(TargetRuntimeBinPath))
-            {
-                throw new Exception($"Requested runtime '{ToolingRuntimeBinPath}' or '{TargetRuntimeBinPath}; does not exist. Please make sure it is installed before running test.");
-            }
+            //TargetRuntimeName = $"dnx-{flavor}-{GetOSPrefix()}-{architecture}.{currentRuntimeVersion}";
+            //TargetRuntimeBinPath = Path.Combine(runtimeHome, TargetRuntimeName, "bin");
 
-            Logger.LogInformation($"Pick target runtime {TargetRuntimeBinPath}");
-            Logger.LogInformation($"Pick tooling runtime {ToolingRuntimeBinPath}");
+            //if (!Directory.Exists(ToolingRuntimeBinPath) || !Directory.Exists(TargetRuntimeBinPath))
+            //{
+            //    throw new Exception($"Requested runtime '{ToolingRuntimeBinPath}' or '{TargetRuntimeBinPath}; does not exist. Please make sure it is installed before running test.");
+            //}
 
-            // Work around win7 search path issues.
-            var newPath = TargetRuntimeBinPath + Path.PathSeparator + Environment.GetEnvironmentVariable("PATH");
-            DeploymentParameters.EnvironmentVariables.Add(new KeyValuePair<string, string>("PATH", newPath));
+            //Logger.LogInformation($"Pick target runtime {TargetRuntimeBinPath}");
+            //Logger.LogInformation($"Pick tooling runtime {ToolingRuntimeBinPath}");
+
+            //// Work around win7 search path issues.
+            //var newPath = TargetRuntimeBinPath + Path.PathSeparator + Environment.GetEnvironmentVariable("PATH");
+            //DeploymentParameters.EnvironmentVariables.Add(new KeyValuePair<string, string>("PATH", newPath));
         }
 
-        protected void DnuPublish(string publishRoot = null)
+        protected void DotnetPublish(string publishRoot = null)
         {
             DeploymentParameters.PublishedApplicationRootPath = publishRoot ?? Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
 
-            var noSource = DeploymentParameters.PublishWithNoSource ? "--no-source" : string.Empty;
             var command = DeploymentParameters.Command ?? "web";
-            var parameters = $"publish {DeploymentParameters.ApplicationPath} -o {DeploymentParameters.PublishedApplicationRootPath}"
-                + $" --runtime {TargetRuntimeName} {noSource} --iis-command {command}";
+            var parameters = $"publish {DeploymentParameters.ApplicationPath}"
```

Comment:
the command variable above should probably be removed. Maybe even DeploymentParameters.Command is not needed anymore?